airbyte-integrations/connectors/source-jira/acceptance-test-config.yml

```diff
@@ -0,0 +1,28 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector/source-acceptance-tests.md)
+# for more information about how to configure these tests
+connector_image: airbyte/source-jira:dev
+tests:
+  spec:
+    - spec_path: "source_jira/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+#  Jira Source has a lot streams. It takes to long to sync them? which caused timeout error
+#  basic_read:
```

Comment:
can we trim down the catalog and test it on a subset of the streams?

Reply:
Yes, I will try to do it.